vars/goMake.groovy

```diff
@@ -10,6 +10,8 @@ def call(body) {
     def ghOrg =  config.githubOrganisation
     def dockerOrg = config.dockerOrganisation
     def prj = config.project
+    def makeCommand = config.makeCommand ?: 'make'
+    def installPreRequisites = config.installPreRequisites
 
```

Comment:
Rather than installing things in the pipelines it's probably better to add it to the builder image.  This means we don't have to install build dependencies on every run which also speeds things up.